launcher/image/fixup_oem.sh

```diff
@@ -26,6 +26,19 @@ main() {
     sed -e 's|.*|[BEGIN_CS_CMDLINE]&[END_CS_CMDLINE]|g'
 
   umount /mnt/disks/efi
+
+  # Now the oem partition is sealed, we mount it to print it's content
+  if [[ ! -d /mnt/disks/oem ]]; then
+    mkdir /mnt/disks/oem
```

Comment:
Cleanup this directory?

Reply:
I think it is fine as this is not get persistent, just a tmp build environment